src/components/App.tsx

```diff
@@ -368,6 +372,9 @@ export class App extends React.Component<any, AppState> {
         if (this.socket) {
           this.socket.off("first-in-room");
         }
+        if (scene) {
+          this.syncActionResult(scene);
+        }
         this.socketInitialized = true;
```

Comment:
I'm thinking we could move the `await loadScene(null)` here as well so encapsulate it.

Reply:
I started with this, but it doesn't work - I think the setState setting isCollaborating to be true causes localStorage to be overwritten, so I saved the values from localStorage earlier.